[2023-05-02T14:50:17.793+0000] {taskinstance.py:1090} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: chatgpt_dag.delete_file scheduled__2023-05-02T14:45:00+00:00 [queued]>
[2023-05-02T14:50:17.797+0000] {taskinstance.py:1090} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: chatgpt_dag.delete_file scheduled__2023-05-02T14:45:00+00:00 [queued]>
[2023-05-02T14:50:17.797+0000] {taskinstance.py:1288} INFO - 
--------------------------------------------------------------------------------
[2023-05-02T14:50:17.797+0000] {taskinstance.py:1289} INFO - Starting attempt 1 of 1
[2023-05-02T14:50:17.797+0000] {taskinstance.py:1290} INFO - 
--------------------------------------------------------------------------------
[2023-05-02T14:50:17.806+0000] {taskinstance.py:1309} INFO - Executing <Task(PythonOperator): delete_file> on 2023-05-02 14:45:00+00:00
[2023-05-02T14:50:17.809+0000] {standard_task_runner.py:55} INFO - Started process 97224 to run task
[2023-05-02T14:50:17.811+0000] {standard_task_runner.py:82} INFO - Running: ['airflow', 'tasks', 'run', 'chatgpt_dag', 'delete_file', 'scheduled__2023-05-02T14:45:00+00:00', '--job-id', '1343', '--raw', '--subdir', 'DAGS_FOLDER/chatgpt_dag.py', '--cfg-path', '/tmp/tmpzieb8ded']
[2023-05-02T14:50:17.811+0000] {standard_task_runner.py:83} INFO - Job 1343: Subtask delete_file
[2023-05-02T14:50:17.841+0000] {task_command.py:389} INFO - Running <TaskInstance: chatgpt_dag.delete_file scheduled__2023-05-02T14:45:00+00:00 [running]> on host trivia-machine.us-central1-a.c.triviapractice-381517.internal
[2023-05-02T14:50:17.877+0000] {taskinstance.py:1516} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=richc
AIRFLOW_CTX_DAG_ID=chatgpt_dag
AIRFLOW_CTX_TASK_ID=delete_file
AIRFLOW_CTX_EXECUTION_DATE=2023-05-02T14:45:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2023-05-02T14:45:00+00:00
[2023-05-02T14:50:17.881+0000] {logging_mixin.py:137} INFO - /home/rich2/airflow/scripts/chatgpt_output/usgeography_2023-05-02_14-50-13.json deleted successfully
[2023-05-02T14:50:17.881+0000] {python.py:177} INFO - Done. Returned value was: None
[2023-05-02T14:50:17.884+0000] {taskinstance.py:1327} INFO - Marking task as SUCCESS. dag_id=chatgpt_dag, task_id=delete_file, execution_date=20230502T144500, start_date=20230502T145017, end_date=20230502T145017
[2023-05-02T14:50:17.902+0000] {local_task_job.py:212} INFO - Task exited with return code 0
[2023-05-02T14:50:17.909+0000] {taskinstance.py:2596} INFO - 0 downstream tasks scheduled from follow-on schedule check
